roles/endpoints/defaults/main.yml

```diff
@@ -238,6 +238,6 @@ endpoints:
       backend_api: 8043
       haproxy_api: 8042
     haproxy:
-      health_check: 'httpchk GET /healthcheck'
+      health_check: 'httpchk /'
```

Comment:
Why did this get changed? we'd like to use the health check middleware. If this isn't working as is, I'd rather we added the middleware to aodh then to just hit `/`.